ql/src/test/results/clientpositive/llap/hybridgrace_hashjoin_2.q.out

```diff
@@ -1421,7 +1421,7 @@ STAGE PLANS:
                           outputColumnNames: _col1
                           input vertices:
                             1 Map 5
-                          Statistics: Num rows: 25 Data size: 2225 Basic stats: COMPLETE Column stats: COMPLETE
+                          Statistics: Num rows: 4 Data size: 356 Basic stats: COMPLETE Column stats: COMPLETE
```

Comment:
I wonder why stats estimation changed

Reply:
Good catch, thanks! This is unexpected indeed... Taking a look now.